modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/data_set.jsp

```diff
@@ -29,6 +29,8 @@ renderResponse.setTitle(ParamUtil.getString(request, "dataSetLabel"));
 			"fdsFilterClientExtensions", fdsAdminDisplayContext.getFDSFilterCETsJSONArray()
 		).put(
 			"namespace", liferayPortletResponse.getNamespace()
+		).put(
+			"resolvedRESTSchemas", fdsAdminDisplayContext.getRESTApplicationResolvedSchemasJSONArray()
```

Comment:
A bug, disguising as SF error. Handling of endpoints with resolved params didn't work with selection filters. Another one to the "saved by TS" pile.